src/Trace/Integrations/Eloquent.php

```diff
@@ -77,12 +77,9 @@ public static function load()
         });
 
         // public function delete()
-        opencensus_trace_method(Model::class, 'delete', function ($model, $query) {
```

Comment:
This was broken and caught by integration tests - there is no parameter for query.